packages/react-native-sdk/src/icons/TopViewBackground.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { StyleProp, ViewStyle } from 'react-native';
+import {
+  Svg,
+  Rect,
+  Defs,
+  LinearGradient,
+  Stop,
+  NumberProp,
+} from 'react-native-svg';
+
+type Props = {
+  /**
+   * Height of the SVG container
+   */
+  height?: NumberProp;
+  /**
+   * Width of the SVG container
+   */
+  width?: NumberProp;
+  /**
+   * Style to be applied to the SVG container
+   */
+  style?: StyleProp<ViewStyle>;
+};
```

Comment:
```suggestion
};
```

unnecessarily would cause a bug in my view, lets just scope it to having width and height for now

Reply:
The status bar height can create a problem in most devices for iOS. Passing style from the app would help here.

![simulator_screenshot_11FF028D-4658-4585-BFE3-E22060B345C1](https://github.com/GetStream/stream-video-js/assets/39884168/4e0226d0-ea8b-40ed-9a91-2652a0298251)

Can be solved using Statusbar.currentHeight though

Reply:
Done ✅